nltk/draw/table.py

```diff
@@ -76,13 +76,8 @@ def __init__(self, master, columns, column_weights=None, cnf={}, **kw):
         :param cnf, kw: Configuration parameters for this widget.
             Use ``label_*`` to configure all labels; and ``listbox_*``
             to configure all listboxes.  E.g.:
-                >>> from nltk.draw.util import TextWidget, CanvasFrame, SpaceWidget
-                >>> from nltk.draw.table import MultiListbox
-                >>> cf = CanvasFrame(closeenough=10, width=300, height=300)
-                >>> c = cf.canvas()
-                >>> master = TextWidget(c, "hiya there", draggable=1)
-                >>> master = SpaceWidget(c, 0, 30)
-                >>> mlb = MultiListbox(master, 5, label_foreground='red')
+                >>> root = Tk()  # doctest: +SKIP
+                >>> MultiListbox(root, ["Subject", "Sender", "Date"], label_foreground='red').pack()  # doctest: +SKIP
```

Comment:
The old code didn't really make sense: I thought `master` or `root` was always a `tkinter.Tk()` instance. Beyond that, it didn't compile I believe.

I also have to skip these, as `Tk()` fails on a device without a display.